examples/client/getBlockFinalizationSummary.ts

```diff
@@ -0,0 +1,64 @@
+import { BlockFinalizationSummary } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Options
+    --help,     -h  Displays this message
+    --block,    -b  A block to query from, defaults to last final block
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+            block: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Get the summary of the finalization data in a given block. Only finalized
+/// blocks will return a finalization summary, if the summary is requested for
+/// a non-finalized block, this will return an object with only the tag field,
+/// with value "none".  If a blockhash is not supplied it will pick the latest
+/// finalized block.
```

Comment:
Triple slash comments are not docstrings in typescript (I assume you have this from rust?), but is actually a format reserved for type references. I think we should use proper JS block comments instead

```ts
/**
 * Block comment goes here
 */
```

Same goes for all other files in the PR

Reply:
I knew triple slashes was not "supported" but I still wanted to differentiate between documentation and comments, I did not know triple slashes were reserved however. Using the JS blocks are better.

I'll `sed` it in #164. I'll change all imports to only be `node-sdk` for these client examples in that same PR.

Reply:
Done now